src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,15 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePage {
+    WebDriver browser;
+    WebDriverWait wait;
+    public BasePage(WebDriver browser) {
+        this.browser = browser;
+        wait = new WebDriverWait(browser, Duration.ofMillis(1000));
```

Comment:
All numbers other than 0 and 1 must be placed in variables, constants or resource files